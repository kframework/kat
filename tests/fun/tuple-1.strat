a49cb448ea6b71ca038e922d9d7f6c61 orig
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx2 exec
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx3 compile
