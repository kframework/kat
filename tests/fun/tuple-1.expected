
Strategy: orig
================================================================================
<FUN>
  <k>
    Pair ( Pair 1 ( Wrapper 2 ) ) ( Pair 3 4 )
  </k>
  <env>
    .Map
  </env>
  <envs>
    .List
  </envs>
</FUN>

Strategy: exec
================================================================================
<kat-FUN>
  <s>
    #STUCK ( )
  </s>
  <kat>
    <analysis>
      .Analysis
    </analysis>
    <states>
      .States
    </states>
  </kat>
  <harness>
    <FUN>
      <k>
        Pair ( Pair 1 ( Wrapper 2 ) ) ( Pair 3 4 )
      </k>
      <env>
        .Map
      </env>
      <envs>
        .List
      </envs>
    </FUN>
  </harness>
</kat-FUN>

Strategy: compile
================================================================================
<kat-FUN>
  <s>
    #STUCK ( )
  </s>
  <kat>
    <analysis>
      .Rules , < <FUN>
        <k>
          let ( x = 1 ) and y = Wrapper 2 and z = Pair 3 4 and .Bindings in Pair ( Pair x y ) z
        </k>
        <env>
          .Map
        </env>
        <envs>
          .List
        </envs>
      </FUN> --> <FUN>
        <k>
          Pair ( Pair 1 ( Wrapper 2 ) ) Pair 3 4
        </k>
        <env>
          .Map
        </env>
        <envs>
          .List
        </envs>
      </FUN> requires #True >
    </analysis>
    <states>
      .States
    </states>
  </kat>
  <harness>
    <FUN>
      <k>
        Pair ( Pair 1 ( Wrapper 2 ) ) Pair 3 4
      </k>
      <env>
        .Map
      </env>
      <envs>
        .List
      </envs>
    </FUN>
  </harness>
</kat-FUN>
